pennylane/queuing.py

```diff
@@ -220,6 +221,9 @@ def _append(self, obj, **kwargs):
     def _remove(self, obj):
         self.queue.remove(obj)
 
+    append = _append
+    remove = _remove
```

Comment:
Trying to understand this one?

Reply:
Ah, this one is weird. `QueuingContext.append` is a _class_ method, so doesn't take self into account. Here, we are in a subclass, and simply overriding the inherited class methods.